### DISCUSSION TITLE
Application error Using in Heroku

### DISCUSSION THREAD

**Discussion by kamaltej382** [scivision/PyLivestream#93] [Q&A]
@scivision 

The code is not working while using in heroku Showing "An error occurred in the application and your page could not be served. If you are the application owner, [check your logs for details](https://devcenter.heroku.com/articles/logging#view-logs). You can do this from the Heroku CLI with the command"  &  "Application error"

How to solve this ??
Please Help !!

**Comment by scivision** [comment]
This sounds like a very generic error; I wouldn't know offhand. I.e. it could be something not working in general with the Python install or ...
